CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,9 @@ Section headings should be at level 3 (e.g. `### Added`).
 
 ## Unreleased
 
+### Added
+- Added `data_is_not_path` flag to skip file checks when initializing `wandb.Html` with a sting that points to a file.
```

Comment:
Can you make this consistent with the usual format and check for typos?

Also, this skips file checks regardless of what kind of content it's given, not only when given a string matching a path to an existing file. I would describe it as "a flag to never interpret the `data` parameter to `wandb.Html` as a path".